docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Builder stage
+FROM node:lts-bookworm AS oid4vc-demo-builder
+SHELL ["/bin/bash", "-c"]
+RUN npm -g install pnpm && SHELL=bash pnpm setup && source /root/.bashrc && pnpm add -g pnpm
+RUN pnpm add typescript tslib
+WORKDIR /opt/oid4vc-demo
+
+COPY /. .
+RUN rm -rf /root/.local/share/pnpm/store
+RUN pnpm install -r
+RUN pnpm build
+
+# Final stage
+FROM node:lts-bookworm
+ARG PACKAGE_PATH
```

Comment:
Hmzz, now we have a Dockerfile tightly coupled to the compose file, which is hard to use standalone. Let's make a ticket to improve this situation and simply have 3 images again, where we remove the folders that are not needed

Reply:
Not coupled to docker-compose perse. This will also do:
docker build --build-arg="PACKAGE_PATH=packages/oid4vci-demo-frontend" --build-arg="NODE_SCRIPT=start:prod"  .

Reply:
Of course, I can read

Reply:
It should at least be documented, as for instance we have a partner that only wants to run the agent

Reply:
tested & documented at the end of the readme